src/minimal-live-connect.ts

```diff
@@ -1,55 +1,69 @@
-// @ts-nocheck
-/* eslint-disable */
-import { mergeObjects } from './pixel/fiddler'
 import { IdentityResolver } from './idex'
-import { enrich as peopleVerified } from './enrichers/people-verified'
-import { enrich as additionalIdentifiers } from './enrichers/identifiers-nohash'
-import { enrich as privacyConfig } from './enrichers/privacy-config'
+import { enrichPrivacyMode } from './enrichers/privacy-config'
 import { removeInvalidPairs } from './config-validators/remove-invalid-pairs'
-import { WrappedReadOnlyStorageHandler } from './handlers/storage-handler'
-import { WrappedCallHandler } from './handlers/call-handler'
-import { StorageStrategies } from './model/storage-strategy'
 import { EventBus, ILiveConnect, LiveConnectConfig } from './types'
 import { LocalEventBus } from './events/event-bus'
 import { ReadOnlyStorageHandler, CallHandler, isObject } from 'live-connect-common'
+import { enrichStorageStrategy } from './enrichers/storage-strategy'
+import { enrichPeopleVerifiedId } from './enrichers/people-verified-id'
+import { WrappedReadOnlyStorageHandler } from './handlers/storage-handler'
+import { WrappedCallHandler } from './handlers/call-handler'
+import { enrichIdentifiers } from './enrichers/identifiers-nohash'
 
+// @ts-ignore
 function _minimalInitialization(liveConnectConfig: LiveConnectConfig, externalStorageHandler: ReadOnlyStorageHandler, externalCallHandler: CallHandler, eventBus: EventBus, push: (event: unknown) => void): ILiveConnect {
   try {
+    const validLiveConnectConfig = {
+      ...removeInvalidPairs(liveConnectConfig, eventBus),
+      identifiersToResolve: liveConnectConfig.identifiersToResolve || []
+    }
+
+    const stateWithStorage =
+      enrichStorageStrategy(enrichPrivacyMode(validLiveConnectConfig))
+
+    const storageHandler = WrappedReadOnlyStorageHandler.make(stateWithStorage.storageStrategy, externalStorageHandler, eventBus)
```

Comment:
Why not make `StorageHandler` & `CallHandler` part of the state as well? Create them with appropriate enrichers. All the rest enrichers will need these objects already initialized.

Reply:
The problem is that we currently sometimes serialize the whole state to json (in the fiddler).
This will fail for them

Reply:
Why would you serialize the state in fiddler?